surfactant/sbomtypes/_contextentry.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC
+# See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from typing import List, Optional
+
+
+@dataclass
+class ContextEntry:
+    extractPaths: List[str]
+    archive: Optional[str] = None
+    installPrefix: Optional[str] = None
```

Comment:
I'm thinking instead of having this under `sbomtypes`, let's create something like a `surfactant/utilities.py`, `surfactant/helpers.py`, `surfactant/config.py` or `surfactant/context.py` file that can have some useful data types like this that aren't part of the SBOM data, but could be useful in several places.

Something like `config.py` could eventually tie in with providing more user configurable settings and letting plugins define custom options to control their behavior.